src/displayapp/DisplayAppRecovery.cpp

```diff
@@ -2,6 +2,7 @@
 #include <FreeRTOS.h>
 #include <task.h>
 #include <libraries/log/nrf_log.h>
+#include <components/fs/FS.h>
```

Comment:
including our headers should be done with `""`, excluding other libs (like lvgl, nrf, std-lib) should be done with `<>`

```suggestion
#include "components/fs/FS.h"
```

Reply:
Done